src/ResourceManager/Resources/Commands.Resources/RoleAssignments/ValidateScope.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.WindowsAzure.Commands.Common
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    public class ValidateScope : ValidateArgumentsAttribute
+    {
+        protected override void Validate(object arguments, EngineIntrinsics engineIntrinsics)
+        {
+            if (arguments != null)
+            {
+                string scope = (string)arguments;
+                var pairs = scope.Split('/');
```

Comment:
I'm not sure if I got your comment, can you explain a little bit more (after iteration#2)?

---
In reply to: [118140390](https://github.com/Azure/azure-powershell/pull/4019#discussion_r118140390) [](ancestors = 118140390)